src/main/java/pacman/Creator.java

```diff
@@ -0,0 +1,20 @@
+import java.util.ArrayList;
+public class Creator {
+
+    private static final int CANT_GHOST = 4;
+    public static final int POS_GHOST_1_X = 30;
+    public static final int POS_GHOST_2_Y = 180;
+    public static final int POS_GHOST_3_Y = 60;
+    public static final int POS_GHOST_4_X = 210;
+    private ArrayList<Ghost> ghosts = new ArrayList<Ghost>();
+    public ArrayList<Ghost> createGhots() {
```

Comment:
Please change the name of the method to CreateGhost()

Reply:
agree, but we should keep camel case though: createGhost()